src/helics/cpp98/Core.hpp

```diff
@@ -131,6 +131,15 @@ class Core {
         helicsCoreSetGlobal(core, valueName.c_str(), value.c_str(), hThrowOnError());
     }
 
+    /** add a global alias for an interface
+    @param interfaceName the given name of the interface
+    @param alias the new name by which the interface can be referenced
+    */
+    void addAlias(const std::string& interfaceName, const std::string& alias)
+    {
+        helicsFederateAddAlias(core, interfaceName.c_str(), alias.c_str(), hThrowOnError());
```

Comment:
```suggestion
        helicsCoreAddAlias(core, interfaceName.c_str(), alias.c_str(), hThrowOnError());
```

Reply:
good catch, I suppose eventually we should have checks for every API call in the C++98 API as well as all the others